<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Eigenschaften</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_Port">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox_Port">
         <property name="minimumSize">
          <size>
           <width>69</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_Parity">
         <property name="text">
          <string>Parität:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QComboBox" name="comboBox_Parity">
         <property name="minimumSize">
          <size>
           <width>69</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_Baud">
         <property name="text">
          <string>Baud Rate:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_Baud">
         <property name="minimumSize">
          <size>
           <width>69</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_StopBits">
         <property name="text">
          <string>Stopbits:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QComboBox" name="comboBox_StopBits">
         <property name="minimumSize">
          <size>
           <width>69</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_DataBits">
         <property name="text">
          <string>Datenbits:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_DataBits">
         <property name="minimumSize">
          <size>
           <width>69</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_FlowControl">
         <property name="text">
          <string>Flusssteuerung:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QComboBox" name="comboBox_FlowControl">
         <property name="minimumSize">
          <size>
           <width>69</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="5">
        <widget class="QPushButton" name="pushButton_Connect">
         <property name="text">
          <string>Verbinden</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Graph</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox_PlotData">
         <property name="text">
          <string>Daten plotten</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>149</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_Trigger">
         <property name="text">
          <string>Trigger:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="comboBox_Trigger"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_shownPoints">
         <property name="text">
          <string>Zeitskala [ms]</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QSpinBox" name="spinBox_ShownPoints">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>0</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>5000</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QChartView" name="graphicsView_Graph"/>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QSlider" name="horizontalSlider_ShownPoints">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>5000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Terminal</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox_ShowReceivedData">
         <property name="text">
          <string>Empfangene Daten anzeigen</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QCheckBox" name="checkBox_AutoNewLine">
         <property name="text">
          <string>Automatische Zeilenumbrüche</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="4" colspan="2">
        <widget class="QPlainTextEdit" name="plainTextEdit_Terminal"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_Clear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton_Save">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pushButton_CopyAll">
         <property name="text">
          <string>Copy All</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>81</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLineEdit" name="lineEdit_Send"/>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="pushButton_Send">
         <property name="text">
          <string>Senden</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>772</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
